Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:2.7
+RUN apt update
+RUN apt upgrade -y
+RUN apt install vim -y
+RUN mkdir /sabot
+WORKDIR /sabot
+ADD . /sabot
+RUN pip install -r requirements.txt
+EXPOSE 8000
```

Comment:
Why is there no ENTRYPOINT or CMD in here? bash in a container is not how containers should be used